packages/nimble-blazor/NimbleBlazor/Components/NimbleTextArea.razor.cs

```diff
@@ -29,6 +29,12 @@ public partial class NimbleTextArea : NimbleInputBase<string?>
     [Parameter]
     public string? Placeholder { get; set; }
 
+    [Parameter]
+    public bool? ErrorVisible { get; set; }
```

Comment:
Heh, the two properties that managed to get documented on `NimbleTextField` are `ErrorVisible` and `ErrorText`. Let's at least bring over that documentation here as well (though it would be good to update all of the documentation for the properties while you're here, but I won't demand that).

Reply:
Added docs for all the properties, plus there was a `Size` property that doesn't do anything from what I can tell, so I removed it.